packages/core/src/accordion/accordion-panel.element.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { baseStyles, CdsBaseAccordionPanel } from '@clr/core/internal';
+import { ClarityIcons, angleIcon } from '@clr/core/icon';
+import { html } from 'lit-element';
+import { styles } from './accordion-panel.element.css.js';
+
+ClarityIcons.addIcons(angleIcon);
+
+/**
+ * Web component accordion
+ *
+ * ```typescript
+ * import '@clr/core/accordion/register.js';
+ * ```
+ *
+ * ```html
+ * <cds-accordion>
+ *   <cds-accordion-panel expanded='panel1Expanded' (expandedChange)='panel1Expanded = $event.detail'>
+ *    <cds-accordion-header>Item 1</cds-accordion-header>
+ *     <cds-accordion-content *ngIf='panel1Expanded'>Content 1</cds-accordion-content>
+ *   </cds-accordion-panel>
+ *   <cds-accordion-panel expanded='panel2Expanded' (expandedChange)='panel2Expanded = $event.detail'>
+ *     <cds-accordion-header>Item 2</cds-accordion-header>
+ *     <cds-accordion-content *ngIf='panel2Expanded'>Content 2</cds-accordion-content>
+ *   </cds-accordion-panel>
+ *
+ *   <cds-accordion-panel disabled>
+ *     <cds-accordion-header>Item 3</cds-accordion-header>
+ *     <cds-accordion-content>Content 3</cds-accordion-content>
+ *   </cds-accordion-panel>
+ * </cds-accordion>
+ * ```
+ *
+ * @element cds-accordion-panel
+ * @slot
+ * @slot cds-accordion-header
+ * @slot cds-accordion-content
+ * @event expandedChange - notify when the user has clicked the panel header
+ * @cssprop --border-color
+ * @cssprop --border-width
+ * @cssprop --color
+ * @cssprop --font-size
+ * @cssprop --font-weight
+ * @cssprop --header-background-color
+ * @cssprop --header-color
+ * @cssprop --header-font-size
+ * @cssprop --header-font-weight
+ * @cssprop --header-open-background-color
+ * @cssprop --transition-duration
+ */
+export class CdsAccordionPanel extends CdsBaseAccordionPanel {
+  connectedCallback() {
+    super.connectedCallback();
+    this.setAttribute('slot', 'accordion-panel');
+  }
+
+  private toggle() {
+    this.expanded = !this.expanded;
+    this.expandedChange.emit(this.expanded);
+  }
+
+  render() {
+    return html` <div class="private-host" role="group">
+      <div class="accordion-header${this.expanded ? ' open' : ''}${this.disabled ? ' disabled' : ''}">
+        <button
+          cds-layout="horizontal p-x:md p-y:sm"
+          type="button"
+          @click="${() => this.toggle()}"
+          class="accordion-header-button"
+          ?disabled="${this.disabled}"
+          aria-disabled="${this.disabled}"
+          aria-expanded="${this.expanded}"
+        >
+          <span class="accordion-status" cds-layout="vertical align:vertical-center p-r:sm">
+            <span class="accordion-angle-wrapper">
+              <cds-icon class="accordion-angle" shape="angle" dir="right" size="14"></cds-icon>
```

Comment:
The accordion angle does change direction when open but via transform and not by changing the attribute value. I'll still update to use `direction`. 

Can you tell me what may be missing from a11y point of view? I vetted it against our Angular implementation and also did screenreader testing locally. It's possible I missed something though.

Reply:
An `aria-label` on the icon is probably called for...

Reply:
The icon by itself is not actionable and is somewhat decorative. I've taken a look at couple of other aria accordions as well. Maybe it makes more sense to set `aria-hidden=true` so that it's hidden from SR user?

Reply:
`aria-hidden="true'` makes a lot of sense in that case...